<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mitgliederdaten</string>
  </property>
  <widget class="QLineEdit" name="lE_Vorname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorname</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Nachname</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_Nachname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Straße</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_Strasse">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>PLZ / Ort</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_PLZ">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_Ort">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefon</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_Telefon">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mobil</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_Mobil">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>E-Mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_Email">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Anrede</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cB_Anrede">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>83</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Herr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Frau</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Familie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="cB_whatsapp">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Möchte in Whatsapp-Gruppe</string>
   </property>
   <property name="text">
    <string>WhatsApp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Geburtstag</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dE_geburtstag">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>350</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Alter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_Kalender">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Geburtstag darf im Kalender angezeigt werden</string>
   </property>
   <property name="text">
    <string>in Kalender anzeigen</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Eintritt</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dE_eintritt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Austritt</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dE_austritt">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestorben</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dE_gestorben">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMitgliedNr">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mitglied-Nr.</string>
   </property>
  </widget>
  <widget class="QLabel" name="MitgliedNr">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_telegram">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Möchte in Whatsapp-Gruppe</string>
   </property>
   <property name="text">
    <string>Telegram</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Beitrag</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_beitrag">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_beitrag">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ermitteln</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_beitrag_gruppe">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Schüler</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rentner</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Erwachsener</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bankverbindung</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_iban">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IBAN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>450</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bank</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_bic">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Funktion</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cB_funktion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mitglied</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1. Vorsitzender</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. Vorsitzender</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1. Kassierer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. Kassierer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1. Kassenprüfer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. Kassenprüfer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Platzwart</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flugleiter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jugendleiter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Datenschutzbeauftragter</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pB_OK">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>650</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_Abbrechen">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>650</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
